inc/PassHash.class.php

```diff
@@ -562,11 +672,11 @@ public function hash_sha512($clear, $salt = null, $magic = null) {
             throw new Exception('This PHP installation has no SHA512 support');
         }
         $this->init_salt($salt, 8, false);
-		if(empty($magic)) {
-			return crypt($clear, '$6$'.$salt.'$');
-		}else{
-			return crypt($clear, '$6$'.$magic.'$'.$salt.'$');
-		}
+        if(empty($magic)) {
+            return crypt($clear, '$6$'.$salt.'$');
+        }else{
+            return crypt($clear, '$6$'.$magic.'$'.$salt.'$');
+        }
```

Comment:
again, I wonder about the whitespaces here. Should be all spaces.

Reply:
there were tabs; there are spaces.